.github/workflows/build-viz-state.yml

```
@@ -12,8 +17,8 @@ jobs:
       - uses: actions/checkout@v3
         with:
           repository: runziggurat/crunchy
-      - run: cargo run --release -- -i latest.json -o latest.viz.json
+      - run: cargo run --release -- -f $FILTER -i latest.json -o latest.viz.json
```

This allows us to cover cases where we don't want to do any filtering:
```suggestion
      - if: ${{ inputs.filter != "" }}
        name: Generate filtered state
        run: cargo run --release -- -f {{ inputs.filter }} -i latest.json -o latest.viz.json
      - if: ${{ inputs.filter == "" }}
        name: Generate unfiltered state
        run: cargo run --release -- -i latest.json -o latest.viz.json
```